.github/workflows/ci.yml

```diff
@@ -72,33 +72,24 @@ jobs:
     name: "cargo test | ${{ matrix.os }}"
     steps:
       - uses: actions/checkout@v4
+      - name: "Install Rust toolchain"
+        run: rustup show
 
-      - if: ${{ matrix.os == 'macos' }}
-        name: "Install bootstrap dependencies"
-        run: brew install coreutils
+      - if: ${{ matrix.os != 'windows' }}
+        uses: rui314/setup-mold@v1
 
-      - uses: actions/setup-python@v5
-        with:
-          python-version: "3.12"
+      - uses: Swatinem/rust-cache@v2
 
       - name: "Install required Python versions"
         run: |
-          python -m pip install "zstandard==0.22.0"
-          python scripts/bootstrap/install.py
-
-      - name: "Install Rust toolchain"
-        run: rustup show
+          cargo run -p uv-dev -- fetch-python
+          echo "$PWD/bin" >> $GITHUB_PATH
```

Comment:
I think it actually is working otherwise the tests would fail to find Python 3.12 when a test does not request specific executables (because we no longer setup a default Python in the runner)?

Reply:
We will find `python.exe` in this directory, i'd have to double check, i don't think this applies equally to the other launcher we create, while they work fine on unix

Reply:
Like here are all the links

```
 2024-04-10T14:06:10.657299Z  INFO run: Linked `bin\python@3.10.13` to `bin\cpython-3.10.13-windows-x86_64-none`
2024-04-10T14:06:10.657361Z  INFO run: Linked `bin\python@3.11.7` to `bin\cpython-3.11.7-windows-x86_64-none`
2024-04-10T14:06:10.657382Z  INFO run: Linked `bin\python@3.12.1` to `bin\cpython-3.12.1-windows-x86_64-none`
2024-04-10T14:06:10.657399Z  INFO run: Linked `bin\python@3.8.12` to `bin\cpython-3.8.12-windows-x86_64-none`
2024-04-10T14:06:10.657415Z  INFO run: Linked `bin\python@3.8.18` to `bin\cpython-3.8.18-windows-x86_64-none`
2024-04-10T14:06:10.657431Z  INFO run: Linked `bin\python@3.9.18` to `bin\cpython-3.9.18-windows-x86_64-none`
```

There are no executables in here on Windows

Reply:
Can you add a comment that this is unix only and why we need it there but not on windows?

Reply:
Yeah I'll investigate, it was needed but maybe not anymore.

Reply:
I think i moved all the PATH handling to explicit overwrites when adding windows support